cmd/tailscaled/tailscaled.go

```diff
@@ -437,6 +436,7 @@ func run() error {
 	if err != nil {
 		return fmt.Errorf("safesocket.Listen: %v", err)
 	}
+	defer dialer.Close()
```

Comment:
Out of curiosity, why this change?

Reply:
It was being closed too early, which resulted in the call to LogoutSync not being able to make the network request to control.